koku/reporting/provider/aws/openshift/models.py

```diff
@@ -98,8 +98,9 @@ class Meta:
     blended_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
     markup_cost_blended = models.DecimalField(max_digits=33, decimal_places=15, null=True)
     savingsplan_effective_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
-    calculated_amortized_cost = models.DecimalField(max_digits=33, decimal_places=9, null=True)
     markup_cost_savingsplan = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    calculated_amortized_cost = models.DecimalField(max_digits=33, decimal_places=9, null=True)
+    markup_cost_amortized = models.DecimalField(max_digits=33, decimal_places=9, null=True)
```

Comment:
You did awesome updating the AWS UI Summary Table SQL. Since were adding these columns to OCP on AWS, we gotta do the same thing in that SQL too. Only thing to keep in mind on this one is its trino SQL and not postgres.

Reply:
I _think_ I updated the correct SQL files.